bench/wtperf/runners/truncate-btree-workload.wtperf

```diff
@@ -0,0 +1,10 @@
+# truncate workload. work phase
+conn_config="cache_size=2GB,checkpoint=(wait=60)"
+table_config="type=file"
+create=false
+# Start with a small set of inserts in the populate phase.
+report_interval=5
+run_time=500
+# Setup three threads to insert into the oplog (will currently work)
+# Setup one thread to be doing truncates from the oplog (not implemented yet)
```

Comment:
The parts of this comment in parentheses are no longer necessary. Similarly for other config files.